Ontology.ttl

```diff
@@ -31,15 +31,16 @@ ids:
        <https://github.com/sebbader> ,
        <https://github.com/Maboeckmann> ,
        <https://github.com/clange> ,
-       <https://github.com/HaydarAk> ;
+       <https://github.com/HaydarAk> ,
+       <https://github.com/JohannesLipp> ;
     dct:publisher ids:IDSA ;
     dct:created "2017-09-26"^^xsd:date ;
     dct:modified "2020-04-30"^^xsd:date ;
     owl:versionInfo "3.1.0" ;
     owl:versionIRI "https://w3id.org/idsa/core/3.1.0>" ;
     vann:preferredNamespaceUri "https://w3id.org/idsa/core/" ;
     vann:preferredNamespacePrefix "ids" ;
-    rdfs:seeAlso <https://industrialdataspace.github.io/InformationModel/> ;
+    rdfs:seeAlso <https://github.com/International-Data-Spaces-Association/InformationModel> ;
```

Comment:
@HaydarAk Do we have a Github.io page for the new repo? If so, then we should refer to it here instead.

Reply:
Thank you for the info, we updated it properly.